src/@next/Card/Card.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { ComponentAction } from '../../types/componentAction';
+import { Button, PrimaryButton } from '../Button';
+import { ButtonGroup } from '../ButtonGroup';
+import { Typography } from '../Typography';
+import {
+  StyledCardActionWrapper,
+  StyledCardContainer,
+  StyledCardContentWrapper,
+  StyledCardHeaderSection,
+  StyledCardHeaderSectionHalf,
+  StyledCardHeaderWrapper,
+} from './CardStyle';
+import { Section } from './Section';
+
+export type CardProps = {
+  heading?: string;
+  subheading?: string;
+  children?: React.ReactNode;
+  primaryAction?: ComponentAction;
+  secondaryAction?: ComponentAction;
+  actionsAlignment?: 'left' | 'right';
+};
+
+const CardComponent = React.forwardRef<HTMLDivElement, CardProps>(function Card(
+  {
+    heading,
+    subheading,
+    primaryAction,
+    secondaryAction,
+    actionsAlignment = 'right',
+    children,
+  }: CardProps,
+  ref
+) {
+  const headingMarkup = (
+    <Typography as="div" variant="body2">
```

Comment:
If I use <FormattedMessage /> component here, the color will be overriden to `#000`. I am suggesting to add the color prop here.

Another question, is there any case that we need to change the typography for the heading? Let's say to use "body1" instead of "body2"

Reply:
Actually, I can use `formatMessage()` function to prevent the color overriding (since it return a string instead of a React Node).

Reply:
@ganiirsyadi I think we shouldn't design a common component with the specific use case here, need to investigate why FormatMessage overriding the color. Using prop will be hard to override

Reply:
`Another question, is there any case that we need to change the typography for the heading? Let's say to use "body1" instead of "body2"`

It shouldn't, or else the heading will look different on each page